src/ArcanaPrime.sol

```diff
@@ -64,12 +67,14 @@ contract ArcanaPrime is ERC721AQueryable, ERC721ABurnable, Ownable, OperatorFilt
 
     bool public operatorFilteringEnabled;
 
-    uint256 public constant WAR_CHEST_SUPPLY = 512;
     uint256 public constant MAX_ENTITLEMENTS_ALLOWED = 2;
     uint256 public constant MAX_QUANTITY_ALLOWED = 3;
+    uint256 public constant MINT_PRICE = 0.1 ether;
     uint256 public constant MAX_SUPPLY = 10_000;
-    uint256 public constant MINT_PRICE = 0.08 ether;
 
+
+    uint256 public mintSupply = 5888;
+    uint256 public nextUnlockTs = block.timestamp;
```

Comment:
The centralisation risk is still present. If the owner is malicious or compromised, right now the first call to `mintWarChestReserve` can be done any time, with any `_supply` argument. And since the owner controls the `_nextUnlockTs` he can put a timestamp that has already passed.

The risk is not mitigated here, one approach I can propose is starting with 
`uint256 public nextUnlockTs = block.timestamp + 15 days`; or something and then on each `mintWarChestReserve` call add 15 more days to `nextUnlockTs`. Also the `supply` should be capped otherwise all remaining tokens can be minted in the first call.